### DISCUSSION TITLE
How to style with keyboard focus?

### DISCUSSION THREAD

**Discussion by ricardo-valero** [chakra-ui/zag#656] [Q&A]
Similar to `react-aria` how would someone solely style when the element is focused by the keyboard and not by touch or mouse?

Example: 
https://react-spectrum.adobe.com/react-aria/Button.html
When you focus using the keyboard you can see the `data-focus-visible`.

<img width="726" alt="image" src="https://github.com/chakra-ui/zag/assets/55701657/aa1af652-8f76-4ccc-a611-41141554810e">

**Comment by segunadebayo** [comment]
Closing as outdated — Zag now provides `data-focus-visible` attributes on relevant components.
